tests/TestCase/Utility/SecurityTest.php

```diff
@@ -70,6 +71,21 @@ public function testHash()
     }
 
     /**
+     * testInvalidHashTypeException
+     *
+     * @return void
+     */
+    public function testInvalidHashTypeException()
+    {
+        try {
+            Security::hash('test', 'doesnotexist', false);
+            $this->fail('Expected \RuntimeException');
+        } catch (RuntimeException $e) {
+            $this->assertTextContains('The hash type `doesnotexist` was not found. Available algorithms are:', $e->getMessage());
```

Comment:
You could use `@expectedException` and `@expectedExceptionMessage` here to save some lines of code 🏌️

Reply:
@markstory does `@expectedExceptionMessage` it work with changing messages? The message contains a list of available algorithms, depending on the system the test runs on it will be different...